compiler/ballerina-lang/src/main/resources/grammar/BallerinaLexer.g4

```diff
@@ -144,7 +145,13 @@ COLON               : ':' ;
 DOT                 : '.' ;
 COMMA               : ',' ;
 LEFT_BRACE          : '{' ;
-RIGHT_BRACE         : '}' ;
+RIGHT_BRACE         : '}'
+{
+if (inStringTemplate)
```

Comment:
Can we format this properly?

Reply:
I followed the same convention used for some other language grammars. Could you please check the following links?

[1] https://github.com/antlr/grammars-v4/blob/81acf9a3a9482d024a544540e19ad078810fa0c2/csharp/CSharpLexer.g4#L163
[2] https://github.com/antlr/grammars-v4/blob/81acf9a3a9482d024a544540e19ad078810fa0c2/php/PhpLexer.g4#L399

Reply:
Ack.

Reply:
This is java syntax? Can't we use the antlr syntax? i.e:
`{inStringTemplate}? -> popMode`

Reply:
First tried with the above-mentioned syntax. However, it seems ANTLR predicates would not work for this scenario as `-> popMode` is an action that always applied to the whole rule. Hence, referred to the [1] and [2] grammars and came up with this.